pages/_app.tsx

```diff
@@ -1,24 +1,35 @@
 import React from 'react'
 import type { AppProps } from 'next/app'
-import '@royalnavy/fonts'
+import '@defencedigital/fonts'
+import { selectors } from '@defencedigital/design-tokens'
+import { GlobalStyleProvider } from '@defencedigital/react-component-library'
 import { createGlobalStyle } from 'styled-components'
 
-import '../global.scss'
+const { color } = selectors
 
-const GlobalStyle = createGlobalStyle`
+const AppGlobalStyle = createGlobalStyle`
   html {
     scroll-behavior: smooth;
```

Comment:
In ` global.scss` the larger breakpoint font sizes were being overridden to 100%, so with that gone text becaomes larger as the default breakpoints font sizes are used.

Should we add a `font-size: 100% !important;` here (under `html`) for a similar effect?

Reply:
Nice spot I didn't notice that